ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/qml/SearchBar.qml

```diff
@@ -19,7 +19,6 @@ import QtQuick.Controls
 TextField {
     id: searchBar
     rightPadding: height
-    inputMethodHints: Qt.ImhNoPredictiveText
```

Comment:
This seems desirable to keep. We don't want the search field to attempt to auto-fill information. Also, if it starts doing that it could throw a wrench into any existing automation tests that don't expect it.

Are there any indications about why this is causing the problem?

Reply:
The default value for [inputMethodHints](https://doc.qt.io/qt-6.2/qml-qtquick-textinput.html#inputMethodHints-prop) is `Qt.ImhNone` so I don't think it will attempt to auto-fill any information. In my tests with that enum removed it didn't show any predictive text.
As for why this is causing the problem, I do not know. I had Sarat test a different sample (Feature Layer Query) that has a TextField box with this enum and that works fine so this enum is probably not directly to blame. I'll do some more digging on this before committing anything.